backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -138,10 +138,14 @@ func toProto(t *time.Time) (*timestamp.Timestamp, error) {
 
 func (fs *experimentStore) StopExperiments(ctx context.Context, ids []uint64) error {
 	if len(ids) != 1 {
-		// TODO: This API will be renamed to StopExperiment and will take a single ID as a parameter
+		// TODO: This API will be changed to take a single ID as a parameter
 		return errors.New("A single ID must be provided")
 	}
-	sql := `DELETE FROM experiment_run WHERE id = $1`
+	sql :=
+		`UPDATE experiment_run 
+         SET execution_time = tstzrange(lower(execution_time), NOW(), '[]') 
+         WHERE id = $1 AND (lower(execution_time) IS NOT NULL OR clock_timestamp() <@ execution_time)`
```

Comment:
1. shouldn't this be `upper` (as opposed to `lower`)?
1. Can we use now() instead of `clock_timestamp()`. Per https://www.cybertec-postgresql.com/en/postgresql-now-vs-nowtimestamp-vs-clock_timestamp/#:~:text=Keep%20in%20mind%3A%20now(),()%20changes%20all%20the%20time I think that `NOW()` is what we want since the value returned from this function does NOT change within a given SQL transaction (which matters in our case since we call `NOW()` twice)